### PR TITLE
Selecting ES6 by default during project scaffolding

### PR DESCRIPTION
**Pull request by luboskmetko**
I've also tried to rewrite the sample module to ES6

### CODE REVIEW

File: generators/app/templates/scripts/browserify/app.js [hunk 1]
```diff
@@ -2,7 +2,8 @@
   Project: <%= name %>
   Author: <%= author %>
  */
-
-var greet = require('./greeting.js')
+<% if (features.has_babel) { %>
+import { greeting as greet } from './greeting';<% } else { %>
```
Comment (jakub300) [inline generators/app/templates/scripts/browserify/app.js]: Could we use `export default` and and `import { greet }`?

File: generators/app/templates/scripts/browserify/greeting.js [hunk 2]
```diff
@@ -1,17 +1,30 @@
-'use strict';<% if (features.has_jquery) { %>
+<% if (features.has_babel) { %>
```
Comment (thymikee) [inline generators/app/templates/scripts/browserify/greeting.js]: I think we can leave this `'use strict';`

Comment (luboskmetko) [inline generators/app/templates/scripts/browserify/greeting.js]: You meant to leave out for those other cases too?

Comment (thymikee) [inline generators/app/templates/scripts/browserify/greeting.js]: Ok, discard that, I missed it's still available in non-ES6 version.

**Pull request merged**